py/test/unit/selenium/webdriver/remote/test_remote_connection.py

```diff
@@ -33,7 +34,8 @@ def test_get_remote_connection_headers_defaults():
     assert 'Connection' not in headers.keys()
     assert headers.get('Accept') == 'application/json'
     assert headers.get('Content-Type') == 'application/json;charset=UTF-8'
-    assert headers.get('User-Agent') == 'Python http auth'
+    assert headers.get('User-Agent').startswith("selenium/%s (python " % __version__)
+    assert headers.get('User-Agent').split(' ')[-1] in {'windows)', 'mac)', 'linux)'}
```

Comment:
Is there a reason you aren't just asserting the entire `User-Agent`?  The second assertion could allow false positives if the wrong platform sent

Reply:
My reasoning was that asserting the whole thing requires duplicating the code to calculate the platform name, which proves nothing except that that code does whatever it does. This way if on OS X it returns "darwin" or Windows it returns "win32", that issue will be caught when these tests are run on those platforms. I don't feel strongly about it though, happy to go a different route here if you prefer.